fighters/robot/src/acmd/specials.rs

```diff
@@ -1,58 +1,94 @@
 
 use super::*;
 
-#[acmd_script( agent = "robot", script = "game_specials" , category = ACMD_GAME , low_priority)]
+#[acmd_script( agent = "robot", script = "game_specialsstart", category = ACMD_GAME, low_priority )]
+unsafe fn robot_special_s_start_game(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    let boma = fighter.boma();
+    frame(lua_state, 1.0);
+    FT_MOTION_RATE(fighter, 0.1);
+}
+
+#[acmd_script( agent = "robot", script = "game_specialairsstart", category = ACMD_GAME, low_priority )]
+unsafe fn robot_special_air_s_start_game(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    let boma = fighter.boma();
+    frame(lua_state, 1.0);
+    FT_MOTION_RATE(fighter, 0.1);
+}
```

Comment:
please use the formatting for FT_MOTION_RATE which I described in an another comment